source/02-FN/01.ptx

```diff
@@ -576,7 +576,7 @@
 </task>
 </activity>
 
-<remark>
+<remark xml:id="FN01-remark-square-nonfunction">
   Notice that <xref ref="FN1-determine-if-function-from-equation"/> shows that equations with a <m>y^2</m> term generally do not define functions. This is because when you take the square root of a value, there should be two solutions. For example, the square root of <m>4</m> is <m>2</m> and <m>-2</m>. 
```

Comment:
```suggestion
  Notice that <xref ref="FN1-determine-if-function-from-equation"/> shows that equations with a <m>y^2</m> term generally do not define functions. This is because to solve for a squared variable, you must consider both positive and negative inputs. For example, both <m>2^2=4</m> and <m>(-2)^2=4</m>.
```

Reply:
Yeah, I think that's good.  @tdegeorge if you like this, hit "commit suggestion"

Reply:
Oh yes! That's much better! I was trying to write it in a "simple" way and didn't realize I was pigeonholing myself!